### ISSUE TITLE
Unmet dependancies using virtualenv on Kali

### ISSUE THREAD

**Issue by zombietango** [sensepost/UserEnum#1]
Just as a note in case anyone else comes upon this, on my Kali build I needed to add two additional modules beyond what was in the requirements.txt to get the scripts to execute: pycrypto and pyasn1. These are dependencies for impacket and not any of the UserEnum scripts directly, but it seems like pip is not picking them up properly. So in case you are getting errors about missing modules Crypto.Cipher and pyasn1.codec.der those will be the ones you need to add to your virtualenv/system. 

TL;DR: Not an issue with UserEnum, but rather with impacket and pip.

**Comment by randomwalksp** [comment]
Thanks, you may want to bring it to the attention of @asolino over at https://github.com/CoreSecurity/impacket.
